ubuntudesign/developer.ubuntu.com
# Dockerfile for building production docker container

A review of this PR effectively constitutes a review of https://github.com/canonical-webteam/generator-canonical-webteam/pull/66, so you could maybe approve that one too if this one is okay.

QA
--

``` bash
docker build --build-arg COMMIT_ID=`git rev-parse HEAD` --tag dev .
docker run -p 80:80 dev
```

Browse to http://localhost, check the developer site works properly.

Try out all other `./run` commands, check they still work.
